index.html

```diff
@@ -9,18 +9,25 @@ <h1>Akshay Sharma</h1>
         <section id="about">
         <h2>About</h2>
    
-            <p>My name is Akshay. I am very excited to be part of Code the Dream. I am very Impressed with the people involved and the structure of this course.  </p>
-            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga maiores facilis repudiandae ea sed officia dolor hic reprehenderit itaque tempore!</p>
-        </section>
+                <p>My name is Akshay. I am very excited to be part of Code the Dream. I am very Impressed with the people involved and the structure of this course.  </p>
+                
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga maiores facilis repudiandae ea sed officia dolor hic reprehenderit itaque tempore!</p>
+        </section id="experience">
+        
+        <section>
         <h2>Experience</h2>
             <ul>
                 <li>First Citizens Bank</li>
                 <li>Chevron (HCL)</li>
         
             </ul>
+        </section>
+
+        <section id="connect">
         <h2>Connect</h2>
-    <a href='https://github.com/Akshay0327'> Github </a>
-    <a href=" mail to: Akshay.Sharma09@gmail.com"> Akshay.Sharma09@gmail.com </a>
+                <a href='https://github.com/Akshay0327'> Github </a>
+                <a href=" mail to: Akshay.Sharma09@gmail.com"> Akshay.Sharma09@gmail.com </a>
+        </section>
 
 </head>
```

Comment:
head tag represents a header, closing tag should be placed relatively early in the script, any contents should be in the body section